packages/specification/examples/GetMyPrescriptions/Bundle/success-empty.json

```diff
@@ -0,0 +1,16 @@
+{
+    "resourceType": "Bundle",
+    "id": "155e1102-9e71-4f19-9453-b905c0bbc494",
+    "meta": {
+        "lastUpdated": "2022-11-21T14:00:00+00:00"
+    },
+    "type": "searchset",
+    "link": [
```

Comment:
Applies to all our Bundles. This implies we're going to implement pagination, do we maybe want to miss it off?

Reply:
Removed link field. Confirmed by Omar.